code/game/objects/items/weapons/twohanded.dm

```diff
@@ -253,25 +253,8 @@
 
 /obj/item/weapon/twohanded/spear/throw_at(atom/target, range, speed, thrower, spin, flying)
 	spin = FALSE
-	//Find the angle the spear is to be thrown at, then rotate it based on that angle at 45 degree increments
-	var/rotation_value = (Get_Angle(thrower, get_turf(target)))
-	switch(rotation_value)
-		if(337.5 to 360, 0 to 22.5)
-			rotation_value = 0 - current_angle
-		if(22.5 to 67.5)
-			rotation_value = 45 - current_angle
-		if(67.5 to 112.5)
-			rotation_value = 90 - current_angle
-		if(112.5 to 157.5)
-			rotation_value = 135 - current_angle
-		if(157.5 to 202.5)
-			rotation_value = 180 - current_angle
-		if(202.5 to 247.5)
-			rotation_value = 225 - current_angle
-		if(247.5 to 292.5)
-			rotation_value = 270 - current_angle
-		if(292.5 to 337.5)
-			rotation_value = 315 - current_angle
+	//Find the angle the spear is to be thrown at, then rotate it based on that angle
+	var/rotation_value = Get_Angle(thrower, get_turf(target)) - current_angle
```

Comment:
you don't even need this. just current angle = get_angle

Reply:
It's needed because spears are at a 45 degree angle already. If for example a spear was thrown north, the Get_Angle would be 0, and thus the rotation doesn't do anything, leaving the spear still pointing northeast.

Reply:
Oh right, the actual rotation part, honk